lib/topology_docker_openswitch/openswitch.py

```diff
@@ -282,6 +278,66 @@ def notify_post_build(self):
         super(OpenSwitchNode, self).notify_post_build()
         self._setup_system()
 
+    # This function dumps the last "LINES_TO_DUMP" lines from the docker
+    # daemon logs
+    # Docker daemon logs can be gathered by different means depending on
+    # the host OS
+    # For example:
+    #    Ubuntu - /var/log/upstart/docker.log
+    #    CentOS - /var/log/daemon.log | grep docker
+    #    Boot2Docker - /var/log/docker.log
+    #    Debian GNU/Linux - /var/log/daemon.log
+    #    Fedora - journalctl -u docker.service
+    #    Red Hat Enterprise Linux Server - /var/log/messages | grep docker
+    #    OpenSuSE - journalctl -u docker.service
+    # For now we'll cater to Ubuntu & CentOS cases alone.
+    def dump_docker_log_file(self):
```

Comment:
This function is called only once, please consider not defining a function at all but just adding the code where this function is being called. If this is to remain as a function it should not be public.